tests/controllers/excludename/excludename/chainsaw-test.yaml

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Authors of Nimbus
+
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: exclude-names-add-csib
+spec:
+  description: >
+    1. Add dev, staging, prod ns
+    2. Adds a csib for excludeNames with dev, staging
+    3. Update csib for excludeNames with dev, prod, regional
+    4. Add a ns with name management - this ns is not part of the excludeNames
+       Add a ns with name regional - this is part of the excludeNames
```

Comment:
The description should explain the purpose of the test and what it involves.  Currently, it seems like you've included the steps involved in performing the test instead of a general overview.

For e.g,.
https://github.com/5GSEC/nimbus/blob/5a4217460ea4adb01a0a1afab9e4634b7b924e1a/tests/controllers/securityintentbinding/create/chainsaw-test.yaml#L9-L11

Reply:
Will be adding the meaningful description in a separate PR